src/boost_histogram/accumulators.py

```diff
@@ -1,15 +1,23 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function
 
-from ._core.accumulators import Sum, Mean, WeightedSum, WeightedMean
```

Comment:
This is taking a very simple static list, and doing run time manipulations with a function that has more lines than the code it replaces, breaking static analysis. We are also losing any ability to not follow the specific naming scheme in the future.

If we add unit tests for a new type here, that will immediately break if we force to do this.

Reply:
Also, PyBind11 is anything but DRY...

Reply:
Remember, `_core` is monkey-patched for documentation, so everything in it should be explicitly imported. It is also ignored for static analysis, so there again, everything should be explicitly imported. Explicit is better than implicit.

Reply:
This (unrelated to list accumulators) change is also what is breaking Python 2!

Reply:
I don't know what pybind11 has to do with it, and on the contrary, it is a good example for being dry. It is even stated in their docs, that they strongly prefer minimal code to do the work. Minimal code equates avoiding redundancy.

Reply:
Your counter arguments make no sense to me. The code is explicit, explicit in the forwarding and transformation rules. I don't have a problem with not being able to do static analysis here.

Reply:
If you have another solution that allows me to easily add an accumulator without changing the code in several places, then go ahead. For now this is better than it was before.